lib/std/Build.zig

```diff
@@ -1060,6 +1063,10 @@ pub fn addNamedWriteFiles(b: *Build, name: []const u8) *Step.WriteFile {
     return wf;
 }
 
+pub fn addNamedLazyPath(b: *Build, name: []const u8, lp: LazyPath) void {
+    b.named_lazy_paths.put(b.dupe(name), lp) catch @panic("OOM");
```

Comment:
Should this be duped?
```suggestion
    b.named_lazy_paths.put(b.dupe(name), lp.dupe(b)) catch @panic("OOM");
```
That seems to be the general convention in many other parts of the build system API, and makes sense if you assume the writer of build.zig isn't expected/instructed to dupe lazy paths themselves.

Reply:
Whoops, yes it should -- thanks for pointing that out!